{
  "id": "MAL-2025-1756",
  "summary": "Malicious code in ciaoaibot-internal (npm)",
  "details": "\n---\n_-= Per source details. Do not edit below this line.=-_\n",
  "modified": "2025-03-03T13:21:39Z",
  "published": "2025-03-03T13:21:39Z",
  "database_specific": {
    "malicious-packages-origins": [
      {
        "modified_time": "2025-03-03T13:21:39Z",
        "import_time": "2025-03-03T15:06:29.162405112Z",
        "versions": [
          "9.4.9"
        ],
        "id": "RLMA-2025-00712",
        "source": "reversing-labs",
        "sha256": "31f887c4990cb6e0840f82f1d08c06887326015321e920e439b1eeeb9896a0d5"
      }
    ]
  },
  "affected": [
    {
      "package": {
        "name": "ciaoaibot-internal",
        "ecosystem": "npm",
        "purl": "pkg:npm/ciaoaibot-internal"
      },
      "versions": [
        "9.4.9"
      ],
      "database_specific": {
        "source": "https://github.com/ossf/malicious-packages/blob/main/osv/malicious/npm/ciaoaibot-internal/MAL-2025-1756.json"
      }
    }
  ],
  "schema_version": "1.6.0",
  "credits": [
    {
      "name": "ReversingLabs",
      "contact": [
        "https://www.reversinglabs.com"
      ],
      "type": "FINDER"
    }
  ]
}